doc/tech/usecases.md

```diff
@@ -177,13 +177,25 @@ CNTT believes that Edge computing is unique in terms of infrastructure requireme
 
         Note (3) - The potential use cases that may consume services from Edge is the Network Model (V2N) and potentially the V2I (According on how the Infrastructure will be mapped to an Edge level)
 
-- **Use Case #4**
```

Comment:
General comment: the diagram in various use cases are inconsistent

Reply:
Hello pankaj,

What is exactly inconsistent in the diagrams? They are not meant to be identical. 

I always put red frame on the spotted use case.

Reply:
Dear Karim, Dear All,

Here are my suggestions for the proposed edge use-cases and their related figures:

-	Have a common name for Infra, e.g., using consistently “Edge Cloud Infrastructure (NFVI)” or just “Cloud Infrastructure (NFVI)”
-	Consistently use of either APP or AS, and showing the Traffic Steering block even if you have a single CDN NF as in fig.1, or a single V2X APP/AS as in fig.3…
-	In first figure, have the DU/CU next to the Antenna not on top of Cloud Infra, because this is confusing with the vRAN use-case
-	In second figure, keep just the DU/CU next to the Antenna, remove the DU/CU which sit on Cloud Infra, if not, we’ll have questions about a “bastard” vRAN use-case :-)

And, finally, because we can also slice/share the RAN, I suggest renaming those use-cases as follows:
-	Use Case #1 – Edge CDN with eMBB Core Network Slicing
-	Use Case #2 – Edge Private 5G with Core Network Slicing
-	Use Case #3 – Edge Automotive (V2X) with uRLLC Core Network Slicing
-	Use Case #4 – Edge vRAN Deployments

Thanks again.
BR/Lucian

Reply:
> Dear Karim, Dear All,
> 
> Here are my suggestions for the proposed edge use-cases and their related figures:
> 
> * Have a common name for Infra, e.g., using consistently “Edge Cloud Infrastructure (NFVI)” or just “Cloud Infrastructure (NFVI)”
[KR] We can always use "Edge Cloud Infrastructure".

> * Consistently use of either APP or AS, and showing the Traffic Steering block even if you have a single CDN NF as in fig.1, or a single V2X APP/AS as in fig.3…
[KR] The AS is Application Server and this is the standard term used by 3GPP (IMS) to define a platform that commonly host different applications. So, it make sense in V2X Case, to make it AS but I do agree we need to make it simple and as unified as possible. I will change that in the Traffic steering figure.

> * In first figure, have the DU/CU next to the Antenna not on top of Cloud Infra, because this is confusing with the vRAN use-case
> * In second figure, keep just the DU/CU next to the Antenna, remove the DU/CU which sit on Cloud Infra, if not, we’ll have questions about a “bastard” vRAN use-case :-)
[KR] For these two points, I kept the conenction dotted because these two architectures are potential deployment options having the full RAN in the cell site or CU/DU on Edge.
However, this might confuse people so let me update based on your comment.
> 
> And, finally, because we can also slice/share the RAN, I suggest renaming those use-cases as follows:
> 
> * Use Case 1 – Edge CDN with eMBB Core Network Slicing
> * Use Case 2 – Edge Private 5G with Core Network Slicing
> * Use Case 3 – Edge Automotive (V2X) with uRLLC Core Network Slicing
> * Use Case 4 – Edge vRAN Deployments
[KR] Looks fine and descriptive.

  
> Thanks again.
> BR/Lucian

Thanks Lucian for the comments. 
Let me add some responses to the above points.
Kindly find it inline.

Reply:
Thanks Karim, fine for me.